grakn-factory/janus-factory/pom.xml

```diff
@@ -276,6 +276,19 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Big Table Related -->
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-1.0</artifactId>
+            <version>${google-bigtable.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
```

Comment:
If you read above <!-- Big Table Related --> Google bit table. Should be more clear though.